### ISSUE TITLE
Revisit local SVG icons implementation

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#334]
Will need to remove the local SVG icon implementation to match blacklight_icon helper's new behavior and accessibility features.

* [x] remove any inline_svg use
* [x] ~~remove any geoblacklight_icon use (for blacklight_icon)~~

**Comment by ewlarson** [comment]
GBL still uses geoblacklight_icon, so our use can continue:
https://github.com/geoblacklight/geoblacklight/pull/901
